app/src/pages/inside/projectSettingsPageContainer/content/elements/ruleList/ruleItem/ruleItem.jsx

```diff
@@ -60,6 +60,7 @@ export const RuleItem = ({ item, actions, onToggle, disabled, content }) => {
               ))}
             </span>
           )}
+          {customNode}
```

Comment:
Do we need additional `customNode` here?
Can we try to handle this case using `actions`?